src/explore-education-statistics-common/src/modules/find-statistics/components/DataBlockTabs.tsx

```diff
@@ -169,5 +169,7 @@ const DataBlockTabs = ({
 
 export default withLazyLoad(DataBlockTabs, {
   offset: 100,
-  placeholder: ({ dataBlock }) => <span data-testid={testId(dataBlock)} />,
+  placeholder: ({ dataBlock, id }) => (
+    <span id={id} data-testid={testId(dataBlock)} />
```

Comment:
I've added this `id` so that `publish_data.robot` can see this element (the release headlines) and scroll to it.

Now that we've wrapped the entire component with `dynamic` in the frontend, `react-lazyload` can't seem to detect if the lazy loaded component should render or not on initial page load. I suspect this might be due to either:

- Previously always rendering on page load for the release headlines
- Not rendering at the right time to calculate its position on the page relative to the scroll

Either way, this should fix this issue.

Reply:
Awesome, thank you :)